README.md

```diff
@@ -35,55 +39,127 @@ First you need to setup the CoreData stack with a specifc file name. Of course y
+
+__warning__ If your code is in Swift you can either generate the NSManagedObject subclasses in _Swift_ or _Obj-C_.
+
+- _Swift_: add `@objc(ClassName)` before the implementation:
+
+		@objc(Entity)
+		class Entity: NSManagedObject {
+			@NSManaged var createdAt: NSDate?
+			@NSManaged var updatedAt: NSDate?
+		}
+
+- _Obj-C_: import the class header in the bridge-header file.
+
+		#import "Entity.h"
+
+### Simple local database
+
+This part explains how to create and configure a single local database that do not need to be updated from an API.
+In this case there is no need of deprecated entities or automatic updates.
+
+As explained earlier, you first need the data inside a NSDictionary object.
+This data will get parsed and the library will apply a CRUD logic on it.
+
+In each extented class the following methods are **required**:
+
+[+ primaryPredicateWithDictionary:](http://cocoadocs.org/docsets/DKDBManager/0.5.1/Categories/NSManagedObject+DKDBManager.html#//api/name/primaryPredicateWithDictionary:) to create a predicate used in the CRUD process to find the right entity corresponding to the given dictionary.
+
+	+ (NSPredicate *)primaryPredicateWithDictionary:(NSDictionary *)dictionary {
+		// If returns nil then only ONE entity will ever be created and updated.
+		// If returns a `false predicate` then a new entity will always be created.
+		// Otherwise the CRUD process use the entity found by the predicate.
+		return NSPredicate(format: "FALSEPREDICATE");
+	}
+
+[- updateWithDictionary:](http://cocoadocs.org/docsets/DKDBManager/0.5.1/Categories/NSManagedObject+DKDBManager.html#//api/name/updateWithDictionary:) to update the current entity with a given dictionary.
+
+	override public func updateWithDictionary(dictionary: [NSObject : AnyObject]!) {
+        // Update attributes
+        self.id         = GET_NUMBER(dictionary, "id")
+        self.firstName  = GET_STRING(dictionary, "firstName")
+        self.lastName   = GET_STRING(dictionary, "lastName")
+    }
+
+The following **optional** ones are also recommended:
+
+[- description](https://developer.apple.com/library/prerelease/ios/documentation/Cocoa/Reference/Foundation/Classes/NSObject_Class/index.html#//apple_ref/occ/clm/NSObject/description) to improve how the receiving entity is described/logged.
+
+	override var description: String {
+        get {
+            return "\(self.id) : \(self.name)"
+        }
+    }
+
+[+ sortingAttributeName](http://cocoadocs.org/docsets/DKDBManager/0.5.1/Categories/NSManagedObject+DKDBManager.html#//api/name/sortingAttributeName) to specify a default order for the [+ all](http://cocoadocs.org/docsets/DKDBManager/0.5.1/Categories/NSManagedObject+DKDBManager.html#//api/name/all) and and [+ count](http://cocoadocs.org/docsets/DKDBManager/0.5.1/Categories/NSManagedObject+DKDBManager.html#//api/name/count) functions.
+
+    override public class func sortingAttributeName() -> String! {
+        return "order"
```

Comment:
because you return a value that should be in the entity, i would say `return "firstName"` with the comment (`will sort the entity by it's firstName attribute`).